details/details.js

```diff
@@ -0,0 +1,29 @@
+let add = document.getElementById('plus');
+let remove = document.getElementById('minus');
+let price = document.getElementById('originalPice');
+let total = document.getElementById('totalPrice');
```

Comment:
total price initially empty.
make total price initially equals the price of 1 piece.